### PR TITLE
accessibility improvements

### PR DESCRIPTION
**Pull request by skomputer**

### CODE REVIEW

File: src/components/Facet.tsx [hunk 1]
```diff
@@ -3,17 +3,14 @@ import CollectionLink from "./CollectionLink";
...
   render(): JSX.Element {
-    let facetStyle: any = {
-      cursor: "pointer"
-    };
-    if (this.props.facet.active) {
-      facetStyle.backgroundColor = "#ddd";
-    }
+    let facetStyle = this.props.facet.active ? {
+      backgroundColor: "#ddd"
+    } : null;
```
Comment (aslagle) [inline src/components/Facet.tsx]: you lost the pointer

File: src/components/__tests__/Book-test.tsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ let book: BookData = {
 };
 
 describe("Book", () => {
-  it("shows the book cover", () => {
+  it("shows the book cover with alt", () => {
```
Comment (aslagle) [inline src/components/__tests__/Book-test.tsx]: check for the alt?

Comment (skomputer) [inline src/components/__tests__/Book-test.tsx]: we decided to remove the alt, but i forgot to remove it from the test description

File: src/components/Facet.tsx (hunk 1 above)
Comment (skomputer) [inline src/components/Facet.tsx]: does it do anything? pointer is default cursor for text links

Comment (aslagle) [inline src/components/Facet.tsx]: oh ok, I guess they weren't text links at some point.

File: src/components/__tests__/Book-test.tsx (hunk 2 above)
Comment (aslagle) [inline src/components/__tests__/Book-test.tsx]: there were a few places with it in the description

File: src/components/Facet.tsx [hunk 3]
```diff
@@ -3,17 +3,14 @@ import CollectionLink from "./CollectionLink";
...
 
     return (
-      <div style={facetStyle}>
+      <li style={facetStyle}>
```
Comment (aslagle) [inline src/components/Facet.tsx]: should this li be in the facet group too?

**Pull request merged**